lib/src/metta/runner/stdlib/stdlib.metta

```diff
@@ -338,7 +342,7 @@
 (= (switch-internal $atom (($pattern $template) $tail))
```

Comment:
I think it also makes sense to assign the type for the `switch-internal`. I expect type of the `switch-minimal` should work.